File: list-00/c/exercise-03/exercise-03.c
//Aumento salarial

#include <stdio.h>

int main(){

	float salario,percent,aumento;

	printf("Qual o seu salario: ");
	scanf("%f", &salario);
	printf("insira a porcentagem do aumento: ");
	scanf("%f", &percent);

	aumento = salario + (salario * (percent / 100));

	printf("Com este aumento você passa a receber %.3f\n" , aumento);

	return 0;
} 

File: list-00/java/exercise03/exercise03.java
// Faça  um  programa  que  calcule  o  aumento  de  um  salário.  Ele  deve  solicitar  o  valor  do  salário  e  a 
// porcentagem do aumento. Exiba o valor do aumento e do novo salário.

package exercise03;
import java.util.Scanner;

public class exercise03 {
    public static void main(String[] args){
        double aumento, salario, NovoSalario;
        
        Scanner valor = new Scanner(System.in);
        
        System.out.println("Insira o valor de seu salário atualmente: ");
        salario = valor.nextDouble();
        System.out.println("Agora insira quantos porcento será seu aumento: ");
        aumento = valor.nextDouble();
        
        NovoSalario = salario + (salario * aumento / 100);
        
        System.out.println("Seu novo salario passa a ser: "+ NovoSalario);
        
    }
    
}
